plugins/container/go-worker/pkg/container/cri.go

```diff
@@ -520,20 +518,21 @@ func (c *criEngine) Listen(ctx context.Context, wg *sync.WaitGroup) (<-chan even
 				}
 
 				var info event.Info
-				// verbose true to return container.Info
-				ctr, err := c.client.ContainerStatus(ctx, evt.ContainerId, true)
+				returnInfo := true
+				ctr, err := c.client.ContainerStatus(ctx, evt.ContainerId, returnInfo)
 				if err != nil || ctr == nil {
 					info = event.Info{
 						Container: event.Container{
 							Type:        c.runtime,
 							ID:          shortContainerID(evt.ContainerId),
 							FullID:      evt.ContainerId,
 							CreatedTime: nanoSecondsToUnix(evt.CreatedAt),
+							IsPodSandbox: true,
```

Comment:
@deepskyblue86 Am I getting it right that we're deducing it's a sandbox just because the call to ContainerStatus is failing?

Reply:
Exactly. The call fails on a sandbox container (`evt.ContainerId` is the sandbox one, can't have an associated `ContainerStatus`).